quipucords/quipucords/settings.py

```diff
@@ -476,3 +476,38 @@ def app_secret_key_and_path():
 # Old hidden/buried configurations that should be removed or renamed
 MAX_TIMEOUT_ORDERLY_SHUTDOWN = env.int("MAX_TIMEOUT_ORDERLY_SHUTDOWN", 30)
 QUIPUCORDS_MANAGER_HEARTBEAT = env.int("QUIPUCORDS_MANAGER_HEARTBEAT", 60 * 15)
+
+# Defining both local memory and Redis as Django back-end caches.
+#
+# For now, we're keeping the default as local memory cache, this way
+# the application will continue to work without the Celery scan manager enabled.
+# Once quipucords only supports the Celery Scan Manager and Redis, we can then
+# switch the default to be Redis.
+QPC_CACHE_TTL_DEFAULT = env.int("QPC_CACHE_TTL_DEFAULT", default=600)
+
+# For test environments, we only want to communicate with the local memory cache.
+QUIPUCORDS_ENVIRONMENT = env("QUIPUCORDS_ENVIRONMENT", default="production")
```

Comment:
We already have `PRODUCTION = env.bool("PRODUCTION", False)`. Why not use that? Do you expect to have more than just production and test/"not production" in the near future?

Reply:
related https://github.com/quipucords/quipucords/pull/2607/commits/5b59c720947899ddc6c6448d782172dc273606ec#r1556202047

Reply:
Also, if we really do want a new setting for this, I'd favor something much more direct with its meaning like `QUIPUCORDS_ENABLE_REDIS_CACHE`.

Reply:
well prod/dev/test environments (dev like running from bash via make serve which we may want to run with or without redis), but yes, the main  settings are:
- thread scan manager, no redis
- celery scan manager, redis.  (prod/dev)
- celery scan manager, no-redis.  (dev/test)

I'll drop the QUIPUCORDS_ENVIRONMENT to the additional control QUIPUCORDS_ENABLE_REDIS_CACHE (which would default to True if QPC_ENABLE_CELERY_SCAN_MANAGER is True). That should give us all the controls we need (including the override settings we'd use in tests). 🤞